tests/unit/type/zset.tcl

```diff
@@ -308,29 +324,29 @@ start_server {tags {"zset"}} {
             assert_error "*NaN*" {r zincrby myzset -inf abc}
         }
 
-        test {ZADD - Variadic version base case - $encoding} {
+        test "ZADD - Variadic version base case - $encoding" {
```

Comment:
why did you have to change quote type?

Reply:
different PR

Reply:
https://github.com/redis/redis/pull/11803/files